app/controllers/api/base_controller/renderer.rb

```diff
@@ -609,9 +609,9 @@ def determine_include_for_find(klass)
 
       def determine_extra_cols(klass)
         virtual_attributes_for(klass) do |type, attr_name, attr_base|
-          return if attr_base.present?
-          return if virtual_attribute_accessor(type, attr_name)
-          return unless klass.attribute_supported_by_sql?(attr_name)
+          next if attr_base.present?
+          next if virtual_attribute_accessor(type, attr_name)
+          next unless klass.attribute_supported_by_sql?(attr_name)
```

Comment:
wow, weird.  I don't follow what's happening here.  Is this method returning a collection of `attr_name.to_sym` that aren't skipped in the prior 3 conditionals?  Is it returning only the last `attr_name.to_sym`?  

I think this mistake happened because it's unclear it's looping through a list of attributes without an `each`.

Reply:
This can be looked at later, it just looked odd to me so I thought it was worth mentioning

Reply:
@jrafanie while I am not going to "blame you" for why this looks like it does, but you definitely inspired the change:

https://github.com/ManageIQ/manageiq-api/pull/887#discussion_r481410812

Half :trollface:ing

Reply:
Yeah, we can come back to it later.  It could be the wrong abstraction.  :trollface: